stdlib/3.4/asyncio/locks.pyi

```diff
@@ -49,11 +51,11 @@ class Condition(_ContextManagerMixin):
     def notify_all(self) -> None: ...
 
 class Semaphore(_ContextManagerMixin):
-    def __init__(self, value: int = ..., *, loop: Optional[AbstractEventLoop] = ...) -> None: ...
+    def __init__(self, value: int = 1, *, loop: AbstractEventLoop = None) -> None: ...
     def locked(self) -> bool: ...
     @coroutine
     def acquire(self) -> Generator[Any, None, bool]: ...
     def release(self) -> None: ...
 
 class BoundedSemaphore(Semaphore):
-    def __init__(self, value: int = ..., *, loop: Optional[AbstractEventLoop] = ...) -> None: ...
+    def __init__(self, value: int = 1, *, loop: AbstractEventLoop = None) -> None: ...
```

Comment:
Matthias's comment applies here too. We'll likely remove the paragraph in PEP 484 soon that says that setting the default to None implies making the argument Optional.